src/api/app/services/user_service/involved.rb

```diff
@@ -15,6 +15,16 @@ def involved_pkg_and_prj_paginated
       Kaminari.paginate_array(involved_packages_and_projects).page(@page).per(MAX_ITEMS_PER_PAGE)
```

Comment:
@eduardoj one suggestion, maybe call the `sort_involved_items` on the paginated elements, since sorting is memory hungry

Reply:
ahh sorry doesnt work, forget about it....